py/requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy
 pybind11==2.6.2
---extra-index-url https://download.pytorch.org/whl/nightly/cu116
-torch==1.13.0.dev20220921+cu116
-torchvision==0.14.0.dev20220921+cu116
+--extra-index-url https://download.pytorch.org/whl/nightly/cu117
+torch==1.13.0.dev20221006+cu117
```

Comment:
We need to update CI if we are updating this

Reply:
Also the root WORKSPACE and the ci WORKSPACE files

Reply:
Yeah they are in the other PR (upgrade for cuda-11.7). This change is required if anyone is testing this particular PR. So I just left it. Once that PR is merged, we can merge this.